### ISSUE TITLE
FIREFOX only: socket.io "interrupted while the page was loading"

### ISSUE THREAD

**Issue by gulakov** [balderdashy/sails#2955]
The connection to ws://[hsotname]/socket.io/?__sails_io_sdk_version=0.11.0&__sails_io_sdk_platform=browser&__sails_io_sdk_language=javascript&EIO=3&transport=websocket&sid=... was interrupted while the page was loading.

according to this you need to unload it to fix, but I tried that and it still has this error 
http://stackoverflow.com/questions/14140414/websocket-interrupted-while-page-is-loading-on-firefox-for-socket-io?rq=1

**Comment by Dallas62** [comment]
Try to upgrade Socket.io in sails.io.js (minified at the top).
I remember that I had issues like that.

**Comment by sailsbot** [comment]
Thanks for posting, @gulakov.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
